Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -718,27 +718,28 @@ RefPtr<StyleRuleFontFeatureValues> CSSParserImpl::consumeFontFeatureValuesRule(C
     if (m_observerWrapper)
         m_observerWrapper->observer().endRuleBody(m_observerWrapper->endOffset(block));
 
-    // Convert block rules to value (remove duplicates...etc)
-    FontFeatureValuesValue fontFeatureValuesValue;
-    auto upsertTag = [&fontFeatureValuesValue](FontFeatureValuesType type, Tag tag) {
+    // Convert block rules to value (remove duplicate...etc)
+    auto fontFeatureValues = FontFeatureValues::create();
+
+    auto upsertTag = [&fontFeatureValues](FontFeatureValuesType type, Tag tag) {
```

Comment:
I know it was there before, but can we also rename this?

Reply:
Fixed